app/src/main/java/cm/aptoide/pt/home/AdsBundleViewHolder.java

```diff
@@ -0,0 +1,62 @@
+package cm.aptoide.pt.home;
+
+import android.graphics.Rect;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+import cm.aptoide.pt.R;
+import cm.aptoide.pt.dataprovider.model.v2.GetAdsResponse;
+import cm.aptoide.pt.utils.AptoideUtils;
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.List;
+import rx.subjects.PublishSubject;
+
+/**
+ * Created by jdandrade on 13/03/2018.
+ */
+
+class AdsBundleViewHolder extends AppBundleViewHolder {
+  private final TextView bundleTitle;
+  private final Button moreButton;
+  private final RecyclerView appsList;
+  private final AdsInBundleAdapter appsInBundleAdapter;
+  private final PublishSubject<HomeClick> uiEventsListener;
+  private final LinearLayoutManager layoutManager;
```

Comment:
Attribute scope (should be local)

Reply:
done in AppsBundleViewHolder, EditorsBundleViewHolder, AdsBundleViewHolder